src/planning.py

```diff
@@ -353,6 +353,7 @@ def _skeleton_generator(
                 current_node = node
                 for _ in range(sesame_max_policy_guided_rollout):
                     if task.goal.issubset(current_node.atoms):
+                        yield current_node.skeleton, current_node.atoms_sequence
```

Comment:
the PR description doesn't make any mention of the goal, so i'm confused why this only happens inside this if

Reply:
I thought I had mentioned it lol, my bad. Thanks!